api/nakadi-event-bus-api.yaml

```diff
@@ -1147,8 +985,7 @@ definitions:
 
       schema:
         type: object
-        allOf:
-          - $ref: '#/definitions/EventTypeSchema'
+        $ref: '#/definitions/EventTypeSchema'
```

Comment:
This ref style is used in a few places in the pr. The tooling seems generates better output compared to using allOf, and it doesn't change the JSON structure.